public/js/project/utilities/math.js

```diff
@@ -68,6 +68,20 @@ module.exports = {
 		}
 		return validPoints;
 	},
+	getValidPreImageOval: function(matrix) {
+		var validPoints = [],
+				angle = Math.random() * Math.PI,
+				r = (Math.random() * 3) + 1;
```

Comment:
I understand the `+ 1` but why `* 3`.
Not questioning you, just curious for the reasoning behind it.